### DISCUSSION TITLE
Hacktribe on Synth can effectively Sample from AUDIO IN?

### DISCUSSION THREAD

**Discussion by parrucasco** [bangcorrupt/hacktribe#333] [General]
I can´t find any detailed information about this on internet, they only speak about synth and sample capabilities, presets moving from one version to another, but i want to know for sure if installing hacktribe on an electribe synth i will be able to sample from the audio input (and edit/chop/process etc, like in E2s)

im about to buy the synth version because is the one i can afford, but i need to know if i can sample from audio (more oriented to my music)

thanks in advance for the answer

**Comment by bangcorrupt** [comment]
Yes, sampling works the same as with the sampler.

**Reply by parrucasco** [reply]
thanks for the fast reply!
